src/compile.ts

```diff
@@ -593,7 +628,9 @@ export function compile(
         } else if (ts.isPrefixUnaryExpression(node)) {
           if (
             node.operator !== ts.SyntaxKind.ExclamationToken &&
-            node.operator !== ts.SyntaxKind.MinusToken
+            node.operator !== ts.SyntaxKind.MinusToken &&
+            node.operator !== ts.SyntaxKind.MinusMinusToken &&
+            node.operator !== ts.SyntaxKind.PlusPlusToken
```

Comment:
Should this be a constant like we did for binary operators?

Reply:
I actually trimmed down the binary constant now and have them all using the same pattern of  using `ts.tokenToString`. I left the BinaryOp's rewriting of `===` and `!==` to `==` and `!=` for now. The individual transpilers need to fail when an op is unsupported instead of us validating typescript.